rmgpy/rmg/main.py

```diff
@@ -945,6 +948,34 @@ def check_model(self):
                         '\n{2}\n{3}'.format(spc.label, spc2.label, spc.toAdjacencyList(), spc2.toAdjacencyList())
                     )
 
+        # Check all core reactions (in both directions) for collision limit violation
+        violators = []
+        for rxn in self.reactionModel.core.reactions:
+            violator = rxn.check_collision_limit_violation(t_min=self.Tmin, t_max=self.Tmax,
+                                                            p_min=self.Pmin, p_max=self.Pmax)
+            if violator is not None:
+                violators.append(violator)
+        if violators:
+            logging.info("\n")
+            logging.warning("{0} CORE reactions violate the collision rate limit!"
+                            "\nSee the 'collision_rate_violators.log' for details.\n\n".format(len(violators)))
+            with open('collision_rate_violators.log', 'w') as violators_f:
+                violators_f.write('*** Collision rate limit violators report ***\n'
+                                  'The "violation factor" is the ratio of the rate coefficient to the collision limit'
+                                  ' rate at the relevant conditions\n\n')
+                for violator in violators:
+                    violators_f.write('{0}\n{1}\nDirection: {2}\nViolation factor: {3:.2f}\n'
+                                      'Violation condition: {4}\n\n'.format(
+                        violator[0], violator[1], violator[2], violator[3], violator[4]))
+        else:
+            if os.path.isfile('collision_rate_violators.log'):
```

Comment:
I think you could replicate the behavior of RMG.log and rename the existing log regardless of whether or not violators were found, i.e. move this check before `if violators:`. It might also be better to put OLD at the end of the filename so that it appears right next to the new file in the file list.